README.md

```diff
@@ -33,6 +33,14 @@ See examples below:
 {{/let}}
 ```
 
+**Inline use**
+Note: requires Ember 2.0+ (ie. does not support 1.13)
+
+```hbs
+{{let greeting (concat "hello " to)}}
+{{greeting}} - <button {{action (action (mut to) "world")}}>Greet the world!</button>  
+```
```

Comment:
Also, it's actually what 'mut' helper does. It's a shortcut for '{{compute (action (mut path)) value }}'

I will look into marking a stream. It might be possible but will require mutating the stream object.